ibis/expr/api.py

```diff
@@ -4419,6 +4419,11 @@ def _table_view(self):
 
 
 def _table_drop(self, fields):
+
+    if isinstance(fields, str):
+        #  We want to  drop just one attribute.
+        return _table_drop(self, [fields])
```

Comment:
I'd prefer to use `fields = [fields]`, instead of recursively calling the function (which IMHO makes things more difficult to understand, and adds an overheat for no reason.

Also, maybe just personal preference, but seems like it'd make sense to keep the `if not fields:` first, and just check this `if` if fields is present.